web-app/web-frontend/script.js

```diff
@@ -20,7 +20,7 @@ function onDragStart(source, piece, position, orientation) {
 
   // only pick up pieces for the side to move
   if ((game.turn() === 'w' && piece.search(/^b/) !== -1) ||
-      (game.turn() === 'b' && piece.search(/^w/) !== -1)) {
+    (game.turn() === 'b' && piece.search(/^w/) !== -1)) {
```

Comment:
The way you had this aligned before was correct! The second `(game.turn)` condition should align with the one on top so
```js
if ((game.turn() === 'w' && piece.search(/^b/) !== -1) ||
    (game.turn() === 'b' && piece.search(/^w/) !== -1)) {
```